packages/zipkin/src/tracer/index.js

```diff
@@ -51,9 +51,7 @@ class Tracer {
     this._defaultTraceId = this.createRootId();
     this._startTimestamp = now();
     this._startTick = hrtime();
-    if (defaultTags) {
-      this.setTags(defaultTags);
-    }
+    this.recorder.setDefaultTags(defaultTags);
```

Comment:
maybe check for the method `setDefaultTags` or check the type is `BatchRecorder` for now?